File: WebAppAPI/Core/WebAppAPI.Application/DTOs/Order/ListOrder.cs
namespace WebAppAPI.Application.DTOs.Order
{
    public class ListOrder
    {
        public string Id { get; set; }
        public string OrderCode { get; set; }
        public string UserName { get; set; }
        public float TotalPrice { get; set; }
        public DateTime DateCreated { get; set; }
    }
}

File: WebAppClient/src/app/contracts/order/list_order.ts
export class List_Order {
    id: string;
    orderCode: string;
    customerName: string;
    totalPrice: number;
    dateCreated: Date;
    statusId: number;
}


export class List_Order_VM {
    id: string;
    orderCode: string;
    customerName: string;
    totalPrice: string;
    dateCreated: Partial<any>;
    status: OrderStatusInfo;
}
export interface OrderStatusInfo {
    badgeClass: string;
    statusText: string;
}